src/main/java/com/kc/project/rashmiwebservice/RashmiSet.java

```diff
@@ -11,36 +11,25 @@ public int size() {
         return count;
     }
 
-    public void add(int item){
+
+
+    public void add(int item) {
+        for (int j = 0; j < intArr.length; j++) {
+            if (intArr[j] == item) {
+                return;
+            }
+        }
         intArr[count] = item;
         count++;
-        if(count==intArr.length){
+        if (count == intArr.length) {
```

Comment:
When you create the new tempArr, you can populate it here with `System.arrayCopy`